examples/all-clusters-app/all-clusters-common/all-clusters-app.matter

```diff
@@ -369,7 +369,7 @@ server cluster OnOff = 6 {
     kAcceptOnlyWhenOn = 0x1;
   }
 
-  readonly attribute boolean onOff = 0;
+  readonly nosubscribe attribute boolean onOff = 0;
```

Comment:
This is off.  We should never have any `"reportable": false` in our ZAP files.  If ZAP is randomly introducing that, please file bugs on them, but in the meantime just fix it up, please?

Reply:
https://github.com/project-chip/zap/issues/1106

Reply:
Thank you!

If you can, please just search/replace `"reportable": 0` in ZAP files with `"reportable": 1`.

Reply:
I've removed all the reportable: 0 changes now.